apisix/admin/plugins.lua

```diff
@@ -85,16 +102,37 @@ function _M.get(name)
 end
 
 
-function _M.get_plugins_list()
-    local plugins = core.config.local_conf().plugins
+function _M.get_plugins_list(subsystem)
+    local http_plugins
```

Comment:
Could you check the `subsystem` here?

Reply:
`subsystem` is being checked here - https://github.com/apache/apisix/pull/9580/files#diff-00090cde519e7280adbc73e830b92089c4e62a22057c1b7577c77dc1456e72e5R108
I didn't get you.

Reply:
You mean before defining `stream_plugins` variable?

Reply:
@monkeyDluffy6017 The check is already performed here https://github.com/apache/apisix/pull/9580/files#diff-8baac20135bcefd8d1beffe08f44b5271bc1c717c6ab2e720a0e14cb1a02599dR262 before calling this function